mlir/lib/Dialect/SCF/IR/SCF.cpp

```diff
@@ -4237,13 +4237,15 @@ LogicalResult scf::IndexSwitchOp::verify() {
     }
     for (auto [idx, result, operand] :
          llvm::zip(llvm::seq<unsigned>(0, getNumResults()), getResultTypes(),
```

Comment:
nit: while we at here, can we replace `llvm::zip(llvm::seq<unsigned>(0, getNumResults()), getResultTypes(), yield.getOperands())` with `llvm::enumerate(getResultTypes(), yield.getOperands())`?